lib/polyfill/mediasource.js

```diff
@@ -192,7 +194,9 @@ shaka.polyfill.MediaSource = class {
     };
 
     if (window.ManagedMediaSource) {
-      const isTypeSupportedManaged = window.ManagedMediaSource.isTypeSupported;
+      const isTypeSupportedManaged =
+          // eslint-disable-next-line no-restricted-syntax
+          ManagedMediaSource.isTypeSupported.bind(ManagedMediaSource);
```

Comment:
Please do this also for MediaSource.isTypeSupported.  Thanks!

Reply:
Done!